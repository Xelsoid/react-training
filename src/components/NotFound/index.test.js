import React from 'react';
import ComponentExample from './index';

describe('NotFound test', () => {
  it('renders correctly', () => {
    const component = shallow(<ComponentExample />);
    expect(component).toMatchSnapshot();
  });
});
