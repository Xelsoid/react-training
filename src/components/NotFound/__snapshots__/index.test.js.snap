// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound test renders correctly 1`] = `
<div
  className="not-found"
>
  No films found
</div>
`;
